guide/daily-spending-wallet/backup-and-recovery/cloud-backup.md

```diff
@@ -115,37 +117,37 @@ height = 541
 
 {% include tip/open.html color="green" icon="check" label="Do: Inform your Users" %}
 
-Before creating a wallet backup via cloud backup, make sure the user knows what they are about to do. Once the backup is complete, provide a summary and some reassurances before moving on. This is a good place for loading interactions and animation because there will likely be a few seconds between when a user clicks to upload to the cloud provider and when it completes.
+Before creating a wallet backup via cloud backup, make sure the user knows what they are about to do. Once the backup is complete, provide a summary and some reassurances before moving on. Let them know where in their cloud storage they can find their backup, and give the file an understandable name. For example, a filename like `BitcoinWalletBackup_Samantha` conveys far more meaning than a randomly generated number.
 
 {% include tip/close.html %}
 
-### Tradeoffs
+## Tradeoffs
 
 By not making users manually backup, you decrease the risk of self-inflicted loss of funds. Automatic cloud backups put less pressure and responsibility on the user.
 
-While this is an easier way for users to set up a wallet, it brings in the potential threat of malicious third-party access. If a user's cloud account is hacked, their funds could be compromised. This can be mitigated by encrypting the wallet recovery data before sending it to the cloud provider. However, the user could still lose access to funds if they forget their cloud login information or the cloud provider blocks their access.
+While this is an easier way for users to set up a wallet, it brings in the potential threat of malicious third-party access. If a user’s cloud account is hacked, their funds could be compromised. This can be mitigated by encrypting the wallet recovery data before sending it to the cloud provider. However, the user could still lose access to funds if they forget their cloud login information or the cloud provider blocks their access.
 
-By bringing cloud storage providers into the equation, we assume that users trust their providers. This also raises the question of [decentralization]({{ 'guide/getting-started/principles/#decentralization' | relative_url }}): a guiding principle of the bitcoin space and could be considered in conflict with the popular saying, “not your keys, not your coins”.
+By bringing cloud storage providers into the equation, we assume that users trust their providers. This also raises the question of [decentralization]({{ 'guide/getting-started/principles/#decentralization' | relative_url }}), a guiding principle of the bitcoin space, and could be considered in conflict with the popular saying, “not your keys, not your coins.”
```

Comment:
A possible remedy is use multiple backups, like one with the LSP encrypted, one on cloud provider

Reply:
Not a bad idea, but I think it's another level of complexity that's not necessary for this daily spending wallet